packages/react-components/src/components/RichTextEditor/Buttons/Table/RichTextInsertTablePane.tsx

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React, { useMemo } from 'react';
+import { FocusZone, FocusZoneDirection, mergeStyles } from '@fluentui/react';
+import type { IContextualMenuItem, Theme } from '@fluentui/react';
+import {
+  insertTableMenuCellButtonSelectedStyles,
+  insertTableMenuCellButtonStyles,
+  insertTableMenuFocusZone,
+  insertTableMenuTitleStyles
+} from '../../../styles/RichTextEditor.styles';
+import { ColumnRowReplaceString, createKey } from '../../../utils/RichTextTableUtils';
+
+// This file uses RoosterJS React package implementation with updates to UI components and styles.
+const MaxRows = 5;
+const MaxColumns = 5;
+
+/**
+ * @private
+ * Component for the insert table pane
+ */
+export const RichTextInsertTablePane = (props: {
+  theme: Theme;
+  item: IContextualMenuItem;
+  onClick: (e: React.MouseEvent<Element> | React.KeyboardEvent<Element>, item: IContextualMenuItem) => void;
+}): JSX.Element => {
+  const { item, onClick, theme } = props;
+  const [column, setColumn] = React.useState(1);
+  const [row, setRow] = React.useState(1);
+
+  const updateSize = React.useCallback((target?: HTMLElement) => {
+    if (target !== undefined) {
+      const column = parseInt(target.dataset.column ?? '-1');
+      const row = parseInt(target.dataset.row ?? '-1');
+
+      if (column > 0 && column <= MaxColumns && row > 0 && row <= MaxRows) {
```

Comment:
So there is no column = 0?

Reply:
no, I've copied this file (almost everything) from RoosterJS and everything is started from 1. See `const items = React.useMemo(() => {` for more details.

I can update if you think we should start from 0

Reply:
We can leave as 1 if that is how rooster originally wrote it. Just thought it was strange the our range value above was set for -1 and then started at 1.